integration_test/remove_disks_test.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
What is the header for?

Reply:
This was auto-generated while creating this repo so I left it as is.